pkg/agent/openflow/client.go

```diff
@@ -1259,3 +1252,14 @@ func (c *client) SendIGMPQueryPacketOut(
 	packetOutObj := packetOutBuilder.Done()
 	return c.bridge.SendPacketOut(packetOutObj)
 }
+
+func (c *client) InstallBridgeUplinkFlows() error {
+	if runtime.IsWindowsPlatform() || c.connectUplinkToBridge {
+		flows := c.hostBridgeUplinkFlows(*c.nodeConfig.PodIPv4CIDR, cookie.Default)
```

Comment:
What if PodIPv4CIDR is nil? There was a check on Linux platform. Although connectUplinkToBridge supports IPv4 only as of now, not having the check is error prone in the future when it supports IPv6.

Reply:
Updated.